docs/usage/bazel.md

```diff
@@ -5,22 +5,172 @@ description: Bazel dependencies support in Renovate
+If no registries are specified, it will default to the [Bazel Central Registry](https://github.com/bazelbuild/bazel-central-registry).
+If you do not set any custom registries, Renovate defaults to the [Bazel Central Registry](https://github.com/bazelbuild/bazel-central-registry).
+
+Here are some important points about Renovate's Bazel registry searches.
+Renovate will:
+
+- use _all_ `--registry` flag values found in a workspace's `.bazelrc` file
+- use any files that are transitively imported by a `.bazelrc` file
+- only use `--registry` flag values that are not associated with [a configuration](https://bazel.build/run/bazelrc#config)
+- query the registries in the order that they are found in the `.bazelrc` file
+
+#### Example: multiple `.bazelrc` files
+
+In this example, there is a `.bazelrc` file which imports another file called `.registry.bazelrc`.
+Both those files have `--registry` values:
+
+```
+# -------------
+# .bazelrc File
+# -------------
+import .registry.bazelrc
+build --registry=https://raw.githubusercontent.com/bazelbuild/bazel-central-registry/main
+
+# ----------------------
+# .registry.bazelrc File
+# ----------------------
+build --registry=https://example.com/custom_registry
+```
+
+The resulting registry list is:
+
+1. <https://example.com/custom_registry>
+1. <https://raw.githubusercontent.com/bazelbuild/bazel-central-registry/main>
+
+#### Example: Registry entries using Bazel configuration
+
+In this example, a `.bazelrc` file has registry values with and without a configuration:
+
+```
+# -------------
+# .bazelrc File
+# -------------
+build:ci --registry=https://internal.server/custom_registry
+build --registry=https://raw.githubusercontent.com/bazelbuild/bazel-central-registry/main
+```
+
+In this case the `https://internal.server/custom_registry` will be ignored.
+The final registry list is:
+
+1. `<https://raw.githubusercontent.com/bazelbuild/bazel-central-registry/main>`
+
+### Supported Bazel module declarations
+
+#### `bazel_dep`
+
+Renovate will update the `version` value for a [bazel_dep](https://bazel.build/rules/lib/globals/module#bazel_dep) declaration.
+
+```python
+bazel_dep(name = "cgrindel_bazel_starlib", version = "0.15.0")
+```
+
+In the example above, the `0.15.0` will be evaluated against a repository's registries.
+If a new version is found, the value will be replaced with the new version.
+
+#### `git_override`
+
+If Renovate finds a [git_override](https://bazel.build/rules/lib/globals/module#git_override), it will ignore the related `bazel_dep` and evaluate the `commit` value at the specified `remote`.
+
+```python
+bazel_dep(name = "cgrindel_bazel_starlib", version = "0.15.0")
+
+git_override(
+    module_name = "cgrindel_bazel_starlib",
+    commit = "fb47f0e9f7c376a7700fc9fe3319231ae57880df",
+    remote = "https://github.com/cgrindel/bazel-starlib.git",
+)
+```
+
+If there is a more recent commit on the primary branch than the one listed, the `commit` value will be updated.
```

Comment:
```suggestion
If the primary branch has a newer commit than in the list, Renovate will update the `commit` value.
```

Could we go for something like this, where we use simpler words and explain what tool does what?